ui/_locales/en/messages.json

```diff
@@ -93,7 +94,7 @@
       "error": {
         "characterCount": "Must be at least 8 characters",
         "noMatch": "Passwords don’t match",
-        "invalidCharacters": "Password can only contain letters, numbers, and special characters"
+        "invalidCharacters": "Must be at least one number, uppercase, lowercase and special character"
```

Comment:
Change to "Must contain at least one number, uppercase, lowercase and special character"